test/run_test.py

```diff
@@ -449,7 +449,7 @@ def calculate_job_times(reports: List["Report"]) -> Dict[str, float]:
 def pull_job_times_from_S3() -> Dict[str, float]:
     if HAVE_BOTO3:
         ci_job_prefix = get_stripped_CI_job()
-        s3_reports: List["Report"] = get_previous_reports_for_branch('origin/nightly', ci_job_prefix)
+        s3_reports: List["Report"] = get_previous_reports_for_branch('origin/viable/strict', ci_job_prefix)
```

Comment:
why don't we just use master?
Seems like there's a `day_before_commit` constraint already in get_previous_reports_for_branch

Reply:
Because viable/strict usually has no failures, so we'd more likely get correct test times (vs if they failed on master).